tools/spellcheck/correct_words.txt

```diff
@@ -763,6 +784,12 @@ ltnote
 ###################################
 # common technical words {{{
 ###################################
+FHWM
+datasets
+Fortran
+NETLIB
+nonrobust
+rubstifying
```

Comment:
hmm I think it is used in the comments of the original code and is also in the R documentation http://stat.ethz.ch/R-manual/R-patched/library/stats/html/lowess.html